### DISCUSSION TITLE
Adding company logo

### DISCUSSION THREAD

**Discussion by sankethkamathk** [christiansandberg/pytest-reporter-html1#17] [Q&A]
is there a way to add company logo near title of the report (without forking or downloading source code).

I tried modifying source code & then seems to be working

**Comment by christiansandberg** [comment]
Yes you can by creating a template that inherits from this and override the "header" block.

There is some example in the README but I can try to explain more if you can't figure out how to do that.

I have created an internal Python package for our company that overrides the header to add logo and adapting some colors.

**Reply by sankethkamathk** [reply]
thank you for the response @christiansandberg  - I couldnt figure out how to do it without downloading source code. I have the 'pytest-reporter-html1' as part of my requirements.txt
